portfolio/src/main/client/src/components/controls/labeled-input.ts

```diff
@@ -0,0 +1,53 @@
+import {htmlElement, htmlFragment} from '@src/util/html';
+
+export interface LabeledInputOptions {
```

Comment:
add a JSDoc comment describing this interface

Reply:
The interface itself seems self-explanatory, but I added documentation for all of its members.

Reply:
Interfaces still need jsdocs. Anything exported should have a jsdoc.